### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Confirm Selected Features _Color Contrast: In dark Theme, Color Contrast for the list items in the tree view below ‘Azure Toolkit’ checkbox is less than 4.5:1

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3335]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #Java tooling for IntelliJ/Eclipse; #C+E-H
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
**Solution Installed:** The Azure Toolkit for Eclipse 3.17.0

 

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** MAS1.4.3

 

**Repro Steps: ​**
1) Open Java EE - Eclipse application and change the theme to dark in (WindowsàPreferencesàGeneralàappearancesàSelect Dark theme)
2) Tab till Help-->Eclipse Marketplace
3) Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till Install now button and press enter
5) Tab till Azure Toolkit checkbox and expand tree view list items
6) Open Color Contrast and Verify if color contrast ratio for list items in the tree view below ‘Azure Toolkit’ checkbox is less than 4.5:1

 

**Actual Result: ​**
In dark Theme, Color Contrast for the list items in the tree view below ‘Azure Toolkit’ checkbox
is less than 4.5:1

 

**Expected Result: ​**
In dark Theme, Color Contrast for list items in the tree view below ‘Azure Toolkit’ checkbox should be greater than or equal to 4.5:1

 **User Impact: ​**
Users with low vision will not be able to see the controls properly if the color contrast ratio is less than required

​**MAS Reference: ​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![Screenshot (350)](https://user-images.githubusercontent.com/38523250/62347062-0b007c80-b516-11e9-858d-1b663779bf5a.png)

**Comment by Flanker32** [comment]
Hi @Accessibility56 , The eclipse marketplace is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/), here is a screen shot of install AWS Eclipse Toolkit, we could meet the same issue.

![image](https://user-images.githubusercontent.com/12445236/64943986-c0f21100-d89f-11e9-8ad4-8bb49b4d3f55.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium, users with low vision will not be able to see the text properly if the color contrast ratio is low.
- Quality of experience: None, Eclipse provides multi themes and the contrast ratio will be larger than 4.5 in other themes.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks!

**Comment by Accessibility56** [comment]
External Bug is logged: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551102

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749785
